### ISSUE TITLE
Adhere to an international standard for paper metadata

### ISSUE THREAD

**Issue by roo2** [aletheia-foundation/aletheia-app#15]
(Dublin core?)

**Comment by step21** [comment]
There are different standards iirc...

**Comment by robsyme** [comment]
Perhaps something like Journal Article Tag Suite ([JATS](https://jats.nlm.nih.gov/)) could be considered. For Aletheia, the permissive [Journal Archiving and Interchange set](https://jats.nlm.nih.gov/archiving) is probably sufficient.

**Comment by KadeMorton** [comment]
@robsyme good call. My brief research at the beginning of scoping Aletheia flagged Dublin Core as one commonly used global standard, but this is a section I want to rely more on the opinions of people actually publishing in academia to ensure we use a schema that's practical and widely used. 

I'm happy to run with JATS if people feel this is the best way forward. The topic will still remain open for a while, once we start coding up the upload interface that allows metadata submission we will have a call and nut out all the different requirements.

**Comment by step21** [comment]
@KadeMorton tbh, most people publishing have no idea which data formats are used. With us, it all word or pdf. Sometimes latex and then pdf, depending on department. But that is given to publisher or open access provider or whatever. How they process it is generally not known. I think anything that is compatible with PKP OJS is probably good, as compared to looking towards proprietary publishing tools. Personally, I have not seen JATS being very prevalent in social sciences. Take as example policyreview.info a well known/funded open access journal. I really tried, but could not find anything about JATS. Similar for many law or policy publications. (though I didn't conduct an exhaustive study ofc)

**Comment by robsyme** [comment]
We're at danger of conflating two issues here - the first is the data format of the publication itself. I agree with @step21 that we should make it as easy as possible for submitters (Word/PDF). The second issue is how/if we encode metadata (authors, keywords, citations, etc) that accompanies the submission.

**Comment by step21** [comment]
Sorry for that @robsyme 
Of course, internally, Aletheia could/should use whichever is easiest/best. So let's say JATS. When parsing uploads/sources however, it should be able to parse whatever is available, no?
Additionally, to clarify: Quick research suggests, that whereas JATS is a specific xml format, dublin core is more of a schema/vocabulary or standard of which features should be represented in metadata.
For more info see https://www.ncbi.nlm.nih.gov/books/NBK100491/ https://en.wikipedia.org/wiki/Dublin_Core

**Comment by KadeMorton** [comment]
FYI, I am keeping an eye on this discussion but waiting until we are at the implementation stage to get stuck in.

**Comment by KadeMorton** [comment]
We have shifted this issue to Aletheia 2.0 as it's not really an MVP feature.
